nifi-nar-bundles/nifi-jslt-bundle/nifi-jslt-processors/src/main/java/org/apache/nifi/processors/jslt/JSLTTransformJSON.java

```diff
@@ -79,6 +84,9 @@
         + "fails, the original FlowFile is routed to the 'failure' relationship.")
 public class JSLTTransformJSON extends AbstractProcessor {
 
+    public static final AllowableValue APPLY_TRANSFORM_TO_ENTIRE_FLOWFILE = new AllowableValue("Entire FlowFile", "Entire FlowFile", "Entire FlowFile");
+    public static final AllowableValue APPLY_TRANSFORM_TO_EACH_OBJECT = new AllowableValue("Each JSON Object", "Each JSON Object", "Each JSON Object");
```

Comment:
Using an enum implementing DescribedValue can make it easier to compare values later on, so it might be worth changing the options, although this is limited in scope and works as it stands.